testkit/examples/server/index.js

```diff
@@ -0,0 +1,255 @@
+/**
+ * @license
+ * Copyright 2017 The Exonum Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/* eslint-env node,mocha */
+
+const exonum = require('exonum-client')
+const fetch = require('node-fetch')
+const expect = require('chai').expect
+
+// // // // Testkit functions // // // //
+
+const TESTKIT_URL = 'http://127.0.0.1:9000/api/testkit'
+
+async function createBlock (txHashes) {
+  const body = (txHashes === undefined) ? { } : {
+    tx_hashes: txHashes
+  }
+
+  const response = await fetch(TESTKIT_URL + '/v1/blocks', {
+    method: 'POST',
+    headers: { 'Content-Type': 'application/json' },
+    body: JSON.stringify(body)
+  })
+  return response.json()
+}
+
+async function rollbackToHeight (height) {
+  const response = await fetch(TESTKIT_URL + '/v1/blocks/' + (height + 1), {
+    method: 'DELETE'
+  })
+  return response.json()
+}
+
+async function getBlockchainHeight () {
+  let response = await fetch(TESTKIT_URL + '/v1/status')
+  response = await response.json()
+  return response.height
+}
+
+describe('CurrencyService', function () {
+  this.slow(500)
+
+  // // // // Service constants // // // //
```

Comment:
I have little understanding of JavaScript, but such comments still confuse me. Isn't it possible (and reasonable) to split a single file into several modules?